stroyprombeton/tests/tests_views.py

```diff
@@ -307,6 +307,23 @@ def test_total_products_filtered(self):
         total = int(soup.find(id='load-more-products')['data-total-products'])
         self.assertEqual(total, options.count())
 
+    def test_active_options(self):
+        """Category page should contain only options with active related products."""
+        options_qs = (
+            models.Option.objects
+            .bind_fields()
+            .filter_descendants(self.root_category)
```

Comment:
It would be more reliable to use here a category with a number of products less then the maximum per page

Reply:
@artemiy312 , i here use the same sorting as category uses. So, this product always will be the first one. It's not semantical binding of course. But it's enough for integration tests, i think